CMakeLists.txt

```diff
@@ -181,6 +181,12 @@ option(ADD_WINDOWS_ENCLAVE_TESTS "Build Windows enclave tests" OFF)
 # Warning: turning on simulation mode on Windows may cause test failures and random crashes
 option(WIN32_SIMULATION "Windows Simulation Mode" OFF)
 
+# Obtain compiler information of the pre-built tests
```

Comment:
Consider moving this logic into tests/CMakeLists.txt or even tests/libcxx/CMakeLists.txt right now since it's a fairly specific use case.